src/engine/ruleset/integrations/windows/rules/defense-evasion-disabling-windows-logs.yml

```diff
@@ -0,0 +1,58 @@
+name: rule/defense-evasion-disabling-windows-logs/0
+
+metadata:
```

Comment:
Isn't it necessary to add the compatibility section?

Reply:
ditto